### PR TITLE
[fix] [docs] Fixed Use examples in sequence-column-manual.md file.

### PR DESCRIPTION
**Pull request by icchux**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/data-operate/update-delete/sequence-column-manual.md [hunk 1]
```diff
@@ -248,7 +249,7 @@ MySQL [test]> select * from test_table;
 +---------+------------+----------+-------------+---------+
 ```
 
-此时就可以替换表中原有的数据
+此时就可以替换表中原有的数据。最后，指定sequence column是在导入批次数据中对相同key数据选择sequence列值最大的记录Load到Doris表。
```
Comment (spaces-X) [inline docs/zh-CN/docs/data-operate/update-delete/sequence-column-manual.md:252]:
```suggestion
此时就可以替换表中原有的数据。综上，在同批次导入数据中key相同会比较sequence列值，选择值最大的记录导入Doris表中，不同批次之间sequence列不起作用。
```

**Comment by spaces-X** [comment]
Please check if the English version of the markdown file located in `docs/en/docs/data-operate/update-delete/sequence-column-manual.md` also has this problem.

**Comment by icchux** [comment]
> Please check if the English version of the markdown file located in `docs/en/docs/data-operate/update-delete/sequence-column-manual.md` also has this problem.

ok, the en doc also has this problem. nedd i create a new pr?

**Comment by spaces-X** [comment]
> > Please check if the English version of the markdown file located in `docs/en/docs/data-operate/update-delete/sequence-column-manual.md` also has this problem.
> 
> ok, the en doc also has this problem. nedd i create a new pr?

No need, you can just append a new commit to fix the English version md  in the same branch of this pr.

**Comment by icchux** [comment]
> > > Please check if the English version of the markdown file located in `docs/en/docs/data-operate/update-delete/sequence-column-manual.md` also has this problem.
> > 
> > 
> > ok, the en doc also has this problem. nedd i create a new pr?
> 
> No need, you can just append a new commit to fix the English version md in the same branch of this pr.

get！the  doc-fix of english version has been commited.

**Review by spaces-X — COMMENTED** [review]
commit suggestions

**Review by spaces-X — APPROVED** [review]
LGTM

**Pull request merged**